void main(){
  // failing test
  throw 'failing test';
}